baseband/vlbi_base/base.py

```diff
@@ -79,6 +81,11 @@ def __init__(self, fh_raw, header0, unsliced_shape, bps, complex_data,
         self._subset = subset
         self._sample_shape = self._get_sample_shape(subset_wrapints)
 
+    @property
+    def fill_value(self):
+        """Value to use for invalid or missing data (Default value is 0)."""
```

Comment:
I have to leave it as a property of `VLBIStreamBase` since the primary inheritance of `VDIFStreamReader` goes as `VDIFStreamBase` -> `VLBIStreamBase`.

That being said, is it necessarily true we will never use `fill_value` for a writer in the future?